### DISCUSSION TITLE
How to set the server name in Nginx?

### DISCUSSION THREAD

**Discussion by hysonlee** [datenstrom/community#225] [Ask a question] [answered]
My local nginx conf set as below:
```server {
listen 80;
server_name localhost;
...
```
the site running normal fine, but when I visit via IP and click "Edit" the page will goto `http://localhost/edte/` . How do i set the `host_name` value?

Yellow already update to v.0.6.7, PHP version is 7.0.14

many thanks!

**Comment by markseuffert** [comment, marked as answer]
I am afraid to say that I am not a big help with Nginx. Can you test it with Yellow 0.6.9?

You can also try the following [catch-all server block](https://www.nginx.com/resources/wiki/start/topics/examples/server_blocks/) in your Nginx configuration:

```
server {
    listen 80 default_server;
    server_name _;
    ...
}
```

... or use two server blocks, one for your static IP and one for localhost:

```
server {
    listen 80;
    server_name 1.2.3.4;
    ...
}

server {
    listen 80;
    server_name localhost;
    ...
}
```
